code/controllers/ProcessScheduler/core/process.dm

```diff
@@ -34,8 +34,6 @@
 	/**
 	 * Config vars
 	 */
-	// Process name
-	var/name
```

Comment:
...what.

Reply:
Conflicting definition with https://github.com/ParadiseSS13/Paradise/pull/8723/files/f5940155ca23e2fd480d0cd1542509a81364cfe8#diff-10327b088c5dddc32a63536ae064e29eR2

Reply:
Yeah, I know that, it's just a really silly var name in this instance.